chromium_src/chrome/browser/component_updater/widevine_cdm_component_installer.h

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_COMPONENT_UPDATER_WIDEVINE_CDM_COMPONENT_INSTALLER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_COMPONENT_UPDATER_WIDEVINE_CDM_COMPONENT_INSTALLER_H_
+
+#include "brave/components/widevine/static_buildflags.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+
+#if BUILDFLAG(WIDEVINE_ARM64_DLL_FIX)
+
+// Upstream may release a new version of the Arm64 DLL at any time. In this
+// case, we want to update its download URL without having to ship a new browser
+// build. Storing the URL in a feature parameter makes this possible:
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+
+namespace component_updater {
+
+BASE_DECLARE_FEATURE(kBraveWidevineArm64DllFix);
+constexpr base::FeatureParam<std::string> kBraveWidevineArm64DllUrl{
```

Comment:
In this case it's ok to declare it in right in the .cc file in anonymous namespace.

Reply:
@goodov I implemented this in 286f3fa75da31153797fc673d8e5ad56d8cb5968. Wdyt?